packages/eslint-config-airbnb-base/rules/style.js

```diff
@@ -3,531 +3,823 @@ module.exports = {
+    // DISABLE: We can keep this off, we don't do any aliasing of 'this', since we use arrow functions everywhere and those inherit the parent scope
     'consistent-this': 'off',
 
     // enforce newline at the end of file, with no multiple empty lines
+    // KEEP
     'eol-last': ['error', 'always'],
 
     // https://eslint.org/docs/rules/function-call-argument-newline
     // TODO: enable, semver-minor, once eslint v6.2 is required (which is a major)
+    // SITUATIONAL ANALYSIS NEEDED: For functions that have many arguments, I see value in assigning them each a new line
+    // for readability, but overall I don't agree with the rule. I think off/consistent actually may be the best bet here
     'function-call-argument-newline': ['off', 'consistent'],
 
     // enforce spacing between functions and their invocations
     // https://eslint.org/docs/rules/func-call-spacing
+    // KEEP
     'func-call-spacing': ['error', 'never'],
 
     // requires function names to match the name of the variable or property to which they are
     // assigned
     // https://eslint.org/docs/rules/func-name-matching
-    'func-name-matching': ['off', 'always', {
-      includeCommonJSModuleExports: false,
-      considerPropertyDescriptor: true,
-    }],
+    // // KEEP
+    'func-name-matching': [
+      'off',
+      'always',
+      {
+        includeCommonJSModuleExports: false,
+        considerPropertyDescriptor: true,
+      },
+    ],
 
     // require function expressions to have a name
     // https://eslint.org/docs/rules/func-names
+    // KEEP
     'func-names': 'warn',
 
     // enforces use of function declarations or expressions
     // https://eslint.org/docs/rules/func-style
     // TODO: enable
+    // KEEP
     'func-style': ['off', 'expression'],
 
     // enforce consistent line breaks inside function parentheses
     // https://eslint.org/docs/rules/function-paren-newline
+    // KEEP
     'function-paren-newline': ['error', 'consistent'],
 
     // Blacklist certain identifiers to prevent them being used
     // https://eslint.org/docs/rules/id-blacklist
     // TODO: semver-major, remove once eslint v7.4+ is required
+    // KEEP
     'id-blacklist': 'off',
 
     // disallow specified identifiers
     // https://eslint.org/docs/rules/id-denylist
+    // KEEP
     'id-denylist': 'off',
 
     // this option enforces minimum and maximum identifier lengths
     // (variable names, property names etc.)
+    // KEEP
     'id-length': 'off',
 
     // require identifiers to match the provided regular expression
+    // KEEP
     'id-match': 'off',
 
     // Enforce the location of arrow function bodies with implicit returns
     // https://eslint.org/docs/rules/implicit-arrow-linebreak
+    // KEEP
+    // Generally I agree with this rule, if an anonymous function returns an implicit expression, I'd prefer to see that stay on the same line
+    // ex: const bleh = (arg) => someFunc(arg);
```

Comment:
Generally I agree with this rule, if an anonymous function returns an implicit expression, I'd prefer to see that stay on the same line:
```
const bleh = (arg) => someFunc(arg);
```